crates/rspack_core/src/compiler/queue.rs

```diff
@@ -292,6 +292,8 @@ impl WorkerTask for BuildTask {
             compiler_context: CompilerContext {
               options: compiler_options.clone(),
               resolver_factory: resolver_factory.clone(),
+              module: Some(module.identifier()),
+              context: module.as_normal_module().and_then(|m| m.get_context()),
```

Comment:
So many context, it makes me kinda confused. Would you provide a easy-to-understand name or add some comments?